ibis/backends/clickhouse/__init__.py

```diff
@@ -82,7 +82,7 @@ def _from_url(self, url: str, **kwargs) -> BaseBackend:
 
         connect_args = {
             "user": url.username,
-            "password": url.password or "",
+            "password": unquote_plus(url.password) or "",
```

Comment:
the `password` attribute can be `None` depending on the connection string, so there needs to be a `None` check before calling `unquote_plus`